src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -962,6 +1007,21 @@ public async Task GetLatestReleaseViewModel_FiltersComments()
                     }
                 },
                 new()
+                {
+                  Release = release,
+                  ContentSection = new ()
```

Comment:
Think there's a formatting problem here, should be `new()`, but I also see a warning on a missing type specification here, and I think it's because the type is 'non-evident'.

![image](https://user-images.githubusercontent.com/4147126/176170591-788a92e3-2cfd-4c36-a88e-edf21b473a9b.png)